gradle/publishing.gradle

```diff
@@ -57,19 +70,19 @@ publishing {
 
 
 bintray {
-    user = System.getenv('BINTRAY_USER')? System.getenv('BINTRAY_USER') : project.property('bintrayUser')
-    key = System.getenv('BINTRAY_API_KEY') ? System.getenv('BINTRAY_API_KEY') : project.property('bintrayApiKey')
-    override = false
-    publications = ['RadarPublication']
+    user System.getenv('BINTRAY_USER')? System.getenv('BINTRAY_USER') : project.property('bintrayUser')
+    key System.getenv('BINTRAY_API_KEY') ? System.getenv('BINTRAY_API_KEY') : project.property('bintrayApiKey')
+    override false
+    publications 'RadarPublication'
     pkg {
         repo = 'org.radarcns'
         name = 'radar-hdfs-sink-connector'
         userOrg = 'radar-cns'
         licenses = ['Apache-2.0']
-        vcsUrl = 'https://github.com/RADAR-CNS/RADAR-HDFS-Sink-Connector.git'
+        vcsUrl = githubUrl
```

Comment:
does this need .git suffix ? Because some cases it does not work without suffix for example cloning in travis

Reply:
If the git version is recent enough, both with and without the `.git` suffix will work.